src/shared.ts

```diff
@@ -185,15 +185,28 @@ export function maintainScrollVisibility(
 	scrollParent: HTMLElement
 ) {
 	const { offsetHeight, offsetTop } = activeElement
+	const { bottom, top } = activeElement.getBoundingClientRect()
 	const { offsetHeight: parentOffsetHeight, scrollTop } = scrollParent
 
-	const isAbove = offsetTop < scrollTop
-	const isBelow = offsetTop + offsetHeight > scrollTop + parentOffsetHeight
+	const isAboveOfParent = offsetTop < scrollTop
```

Comment:
Awesome, @ikniaziuk! Works great for me! https://deploy-preview-82--vue-accessible-selects.netlify.app/?path=/docs/selectsingle--primary 

It honestly works **so. well.** in that it only scrolls the page if it absolutely needs to. It's perfect!

Last request: can we rename these variables:

`isAboveParent` rather than `isAboveOfParent`
`isBelowScreen` rather than `isBelowOfScreen` etc.

Just reads a bit better. Thanks!

Reply:
sure

Reply:
done